gamestonk_terminal/stocks/sector_industry_analysis/sia_controller.py

```diff
@@ -244,6 +392,7 @@ def print_help(self):
 [param]Country           : [/param]{self.country}
 [param]Market Cap        : [/param]{self.mktcap}
 [param]Exclude Exchanges : [/param]{self.exclude_exchanges}
+[param]Period            : [/param]{self.period}
```

Comment:
You forgot to add the "period" command on the menu so users know they can change it